code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -200,9 +200,9 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		playsound(src, 'sound/items/cigs_lighters/cig_light.ogg', 75, 1, -1)
 		src.reagents.set_temperature(T0C + 45)
 		damtype = "fire"
-		if(REAGENT_VOLUME(reagents, /singleton/reagent/toxin/phoron)) // the phoron explodes when exposed to fire
+		if(REAGENT_VOLUME(reagents, /singleton/reagent/toxin/phoron/base)) // the phoron explodes when exposed to fire
```

Comment:
Same as above. Also, "base" phoron is a pretty bad name - maybe "pure" would work better.

Reply:
Same as above, and changed